src/bundler/LinkerContext.zig

```diff
@@ -252,10 +252,8 @@ pub const LinkerContext = struct {
         reachable: []const Index.Int,
     ) void {
         bun.assert(this.options.source_maps != .none);
-        this.source_maps.line_offset_wait_group.init();
-        this.source_maps.quoted_contents_wait_group.init();
-        this.source_maps.line_offset_wait_group.counter = @as(u32, @truncate(reachable.len));
-        this.source_maps.quoted_contents_wait_group.counter = @as(u32, @truncate(reachable.len));
+        this.source_maps.line_offset_wait_group = .initWithCount(@as(u32, @intCast(reachable.len)));
+        this.source_maps.quoted_contents_wait_group = .initWithCount(@as(u32, @intCast(reachable.len)));
```

Comment:
```suggestion
        this.source_maps.line_offset_wait_group = .initWithCount(reachable.len);
        this.source_maps.quoted_contents_wait_group = .initWithCount(reachable.len);
```

there must've been a reason for these originally, but doesn't seem needed now